doc/getting-started.md

```diff
@@ -370,3 +371,10 @@ print(water.get_positions())
 ```
 [DP]:https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.120.143001
 [DP-SE]:https://dl.acm.org/doi/10.5555/3327345.3327356
```

Comment:
The section `know limitations` should be placed before the references

Reply:
solved